components/ChallengeMaterial/ChallengeMaterial.tsx

```diff
@@ -16,7 +16,7 @@ import dayjs from 'dayjs'
 import relativeTime from 'dayjs/plugin/relativeTime'
 import LocalizedFormat from 'dayjs/plugin/localizedFormat'
 import GiveStarCard from '../GiveStarCard'
-import _ from 'lodash'
+import _, { toNumber } from 'lodash'
```

Comment:
Don't import `toNumber` separately when `_` is already being used to access lodash functions.